### ISSUE TITLE
Allow use of Alumentum in IE Blast Furnace

### ISSUE THREAD

**Issue by Barhandar** [Haggle1996/RevolutionPack#564]

**Comment by Aivech** [comment]
Why?

**Comment by Barhandar** [comment]
Because it's magical coke and because you can use it in RC blast furnace.

**Comment by Eb-** [comment]
I think this is a wonderful idea.

**Comment by AshIndigo** [comment]
I see zero reason to not add  this

**Comment by Kataiser9** [comment]
Yeah, I don't see a problem. It's not like this makes anything easier - charcoal is still the path of least resistance. Besides, adding more utility to Thaumcraft (which is normally kinda weak early game) sounds good.
